packages/issuer-rest/lib/oid4vci-api-functions.ts

```diff
@@ -379,9 +380,16 @@ export function pushedAuthorizationEndpoint<DIDDoc extends object>(
   })
 }
 
+function addTokenEndpoint(issuerMetadata: CredentialIssuerMetadataOpts) {
```

Comment:
setDefaultTokenEndpoint or something, as this method is not for adding token endpoints which the name suggests.

The real question is whether it should be done here, or in the VCIIssuer itself

Reply:
renamed this to the name that you've suggested and moved it to VcIssuerBuilder